modules/tinymce/src/plugins/table/main/ts/actions/TableActions.ts

```diff
@@ -141,6 +143,9 @@ export const TableActions = (editor: Editor, lazyWire: () => ResizeWire, selecti
   const makeColumnsHeader = execute(TableOperations.makeColumnsHeader, Fun.always, Fun.noop, lazyWire, Events.structureModified);
   const unmakeColumnsHeader = execute(TableOperations.unmakeColumnsHeader, Fun.always, Fun.noop, lazyWire, Events.structureModified);
 
+  const makeRowsHeader = execute(TableOperations.makeRowsHeader, Fun.always, Fun.noop, lazyWire, Events.structureModified);
```

Comment:
I'm sorry, but there's a reason we aren't using the `RowsHeader` operations from snooker for this and instead have the custom `setTableRowType` API. It's because snooker doesn't work with the multiple ways you can define a table row header type and it needs to take into account the `table_header_type` setting.

There's a JIRA in the backlog IIRC to make the snooker API work for that, but @metricjs ran out of time for the 5.5 release and we've yet to get back to it. As such this API cannot be used at this time and you'll need to use the other API instead.